### DISCUSSION TITLE
How do I get info about incoming RTMP/SRT?

### DISCUSSION THREAD

**Discussion by vampirefrog** [OvenMediaLabs/OvenMediaEngine#1448] [Help] [answered]
Howdy, I'm sending RTMP or SRT from OBS to OME, and I'm using the REST API to create the vhost and the application and the pushes (restreaming scenario). I have tried `GET /v1/stats/current/vhosts/{vhost}` and `/v1/stats/current/vhosts/{vhost}/apps/{app}` but neither of them give me the information of whether or not RTMP/SRT is being received by OME. They only give me info about what's going out of OME, not what's coming in. How can I get this information? Thank you.

**Comment by getroot** [comment, marked as answer]
Try using the stream API rather than the statistics API.
https://airensoft.gitbook.io/ovenmediaengine/rest-api/v1/virtualhost/application/stream

**Reply by vampirefrog** [reply]
I have tried using the stream API but it comes up with an empty list, even when sending SRT from OBS to OME.

**Reply by vampirefrog** [reply]
What I'm trying to achieve is to only show the llhls player when receiving RTMP/SRT, so I need a way to check whether there is a signal.

**Reply by vampirefrog** [reply]
Thank you, I have figured out how to get the stream info, it requires the stream name that you give it when you :startPush. However, I found it confusing that the stream isn't listed in the `/streams` endpoint.
